pages/account/manage/repos.js

```diff
@@ -136,7 +136,7 @@ export default function ManageRepos({ BASE_URL, repos }) {
           additionalMessage={showNotification.additionalMessage}
         />
 
-        <form onSubmit={handleSubmit}>
+        <form className="mb-4" onSubmit={handleSubmit}>
```

Comment:
I think mb-4 will be too much of space mb-2 might be ideal for this and instead of adding styles on form element  why don't you  add styles in the button element.

Reply:
Directly applying margin style to the button is breaking the button UI.